docs/generalized_distribution.md

```diff
@@ -0,0 +1,83 @@
+# Generalized distribution
+
+As of Zeebe version 8.3 we have introduced a generic way to distribute commands to other partitions.
+We felt the need to generalize this as we are seeing more feature requests that will require
+distribution.
+
+## Why do we need to distribute commands?
+
+Zeebe is a distributed system. We have multiple partitions working together to achieve high
+availability and performance.
+
+When we deploy a new process this process must be available on all
+partitions. If this is not the case we would only be able to start instances of this process on a
+single partition. By distributing it instances can be started across all partitions.
+
+## How do we distribute commands?
+
+On a high-level we use commands and event to distribute commands. The image below provides a nice
+overview of the flow of commands and events that are written during distribution.
+
+![Generalized distribution overview](assets/generalized_distribution.png)
```

Comment:
❌ Let's clarify that the timer start event for redistribution has doubling exponential backoff from 10s to 5m

Reply:
😲 I didn't know that!

Reply:
Although, the timer doesn't have the backoff, correct? We run every 10 seconds and in here we check for each pending distribution if we retries it before?

Reply:
That is correct 👍